diffsims/tests/crystallography/test_get_hkl.py

```diff
@@ -28,12 +28,17 @@
     ReciprocalLatticeVector,
 )
 
+if np.__version__ >= "2.0.0":
```

Comment:
I suggest to add a #TODO comment stating that this check should be removed when NumPy >= 2 is the minimal supported version.

Should be done the other places as well.

Reply:
Actually, this should be >= "1.25.0" because that's when `numpy.exceptions.VisibleDeprecationWarning` was introduced.